examples/algorithms/deepCORAL.py

```diff
@@ -27,9 +27,14 @@ def __init__(self, config, d_out, grouper, loss, metric, n_train_steps):
         assert config.uniform_over_groups
         assert config.distinct_groups
         # initialize models
-        featurizer = initialize_model(config, d_out=None).to(config.device)
-        classifier = torch.nn.Linear(featurizer.d_out, d_out).to(config.device)
-        model = torch.nn.Sequential(featurizer, classifier).to(config.device)
+        if config.model == 'code-gpt-py': #in case of pre-trained language model (`classifier` is also pre-trained)
```

Comment:
It'd be ideal to remove LM-specific code from here, for example by having the initialize_model() take care of initializing (featurizer, classifier)